.github/workflows/release.yml

```diff
@@ -44,8 +44,8 @@ jobs:
     steps:
       - name: Checkout sources
         uses: actions/checkout@v3
-      - name: Get latest version of stable Rust
-        run: rustup update stable
+      - name: Set version of nightly Rust
+        run: rustup default nightly-2023-04-17
```

Comment:
@dutterbutter I can not find the place where the `rustup` is installed for this workflow. Maybe you know, is it preinstalled?